libnetdata/log/log.c

```diff
@@ -908,6 +910,11 @@ void error_limit_int(ERROR_LIMIT *erl, const char *prefix, const char *file __ma
 }
 
 void error_int(int is_collector, const char *prefix, const char *file __maybe_unused, const char *function __maybe_unused, const unsigned long line __maybe_unused, const char *fmt, ... ) {
+#if !defined(NETDATA_INTERNAL_CHECKS) && !defined(NETDATA_DEV_MODE)
+    if (use_severity_level < NETDATA_LOG_LEVEL_ERROR)
+        return;
```

Comment:
Should add similiar check to `info_int()`

Reply:
I added with last commits.